.github/workflows/ci.yaml

```diff
@@ -23,10 +23,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macOS-latest]
-        python-version: [2.7] 
-        #python-version: [2.7, 3.8] 
-        #python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
-        gromacs-version: ["4.6.5", "2018.6", "2020.6", "2021.1"]
+        #python-version: [2.7]
+        #python-version: [2.7, 3.8]
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+        #gromacs-version: ["4.6.5", "2018.6", "2020.6", "2021.1"]
```

Comment:
```suggestion
```

Reply:
Since you changed the matrix to iterate over Python versions, you must *include* tests for the GROMACS versions.